src/app.js

```diff
@@ -1,21 +1,27 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const fs = require('fs');
 
-module.exports = sum;
+const server = http.createServer((req, res) => {
+  const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+
+  const pureFile = normalizedURL.pathname.slice(6) || 'index.html';
+
+  if (!normalizedURL.pathname.startsWith('/file/')) {
+    res.status(404).end(
+      'Please, use /file/ for the correct file path.'
+    );
+  }
+
+  if (fs.existsSync(`./public/${pureFile}`)) {
+    const data = fs.readFileSync(`./public/${pureFile}`, 'utf8');
+
+    res.end(data);
+  } else {
+    res.status(404).end('File not found');
+  }
+
```

Comment:
It's better to wrap these if-else blocks into the try-catch statement and return a 500 error code if an error happened